apps/files/src/components/FilesAppBar.vue

```diff
@@ -213,6 +223,13 @@ export default {
         client: this.$client,
         absolutePath: absolutePath,
         $gettext: this.$gettext
+      }).catch((error) => {
```

Comment:
I removed this from the store to be capable on the application level to deal with the response - see the FileList.vue - in case the response is a 401 the user is redirected to the password page

Reply:
I see. Okay, thanks 👍